src/cascadia/inc/WindowingBehavior.h

```diff
@@ -4,7 +4,8 @@ Licensed under the MIT license.
 --*/
 
 #pragma once
-constexpr int64_t WindowingBehaviorUseCurrent{ 0 };
-constexpr int64_t WindowingBehaviorUseNew{ -1 };
-constexpr int64_t WindowingBehaviorUseExisting{ -2 };
-constexpr int64_t WindowingBehaviorUseAnyExisting{ -3 };
+constexpr int32_t WindowingBehaviorUseCurrent{ 0 };
+constexpr int32_t WindowingBehaviorUseNew{ -1 };
+constexpr int32_t WindowingBehaviorUseExisting{ -2 };
+constexpr int32_t WindowingBehaviorUseAnyExisting{ -3 };
+constexpr int32_t WindowingBehaviorUseName{ -4 };

```

Comment:
**scary**! What if they pass `-w -4`?

Reply:
IIRC, for negative numbers, we figure out that it's a number, then clamp the number to [-1, INT_MAX], then return it, so we never parse `-4` as a string